addon/ng2/tasks/lib-install.js

```diff
@@ -46,14 +45,24 @@ module.exports = Task.extend({
           return this.announceOKCompletion();
         }
 
-        return this.installProcedure(this.autoInjection)
+        if (this.autoInjection) {
+          if (existsSync(path.resolve(process.cwd(), 'src', 'app.ts'))) {
+            var entryPoint = path.resolve(process.cwd(), 'src', 'app.ts');
+            var packageData = this.parseFile(this.packages[0]);
+            this.importInBootstrap(entryPoint, this.packages[0], packageData);
+          }
+
+          return this.announceOKCompletion();
+        }
+
+        return this.installProcedure()
           .then(function(resp) {
             return this.announceOKCompletion();
           }.bind(this));
       }.bind(this));
   },
 
-  installProcedure: function(autoInjection) {
+  installProcedure: function() {
     var allPackages         = {};
```

Comment:
Improved. Thanks for analyzing the code Ciro, if you find anything else please just add more notes.